src/utils/base.test.js

```diff
@@ -0,0 +1,221 @@
+
+  describe('textFont', () => {
+    it('should assign textFont value to fontFamily', () => {
+      const textFontValue = 'Times New Roman';
+      _this.props.textFont = textFontValue;
+      _this.context.styles.fonts = { font: textFontValue };
+      const res = fn();
+      expect(res.fontFamily).toEqual(textFontValue);
+    });
+  });
+
+  describe('fontFamily', () => {
+    it('should assign fontFamily to default font Helvetica', () => {
+      _this.props.textFont = _this.context.styles.fonts.primary;
+      const res = fn();
+      expect(res.fontFamily).toEqual('Helvetica');
+    });
+  });
+
+  // Need to figure out how to test for textSize
+
+  describe('textAlign', () => {
+    it('should assign textAlign value to textAlign', () => {
+      const textAlignValue = 'right';
+      _this.props.textAlign = textAlignValue;
+      const res = fn();
+      expect(res.textAlign).toEqual(textAlignValue);
+    });
+  });
+
+  describe('bgColor', () => {
+    it('should assign bgColor value to backgroundColor ', () => {
+      const bgColorValue = 'blue';
+      _this.props.bgColor = bgColorValue;
+      _this.context.styles.colors = { color: bgColorValue };
+      const res = fn();
+      expect(res.backgroundColor).toEqual(bgColorValue);
+    });
+
+    it('should assign background to default color primary', () => {
+      _this.props.bgColor = _this.context.styles.colors.primary;
+      const res = fn();
+      expect(res.backgroundColor).toEqual('pink');
+    });
+  });
+
+  describe('bgGradient', () => {
+    it('should assign bgGradient value to backgroundImage', () => {
+      const bgGradientValue = 'radial-gradient(red blue green)';
+      _this.props.bgGradient = bgGradientValue;
+      const res = fn();
+      expect(res.backgroundImage).toEqual(bgGradientValue);
+    });
+  });
+
+  describe('bgImage', () => {
+    it('should assign bgImage value to backgroundImage', () => {
+      const bgImageValue = 'https://sameimage.com/url....';
+      _this.props.bgImage = bgImageValue;
+      const res = fn();
+      expect(res.backgroundImage).toEqual(`url(${bgImageValue})`);
+    });
+  });
+
+  describe('bgImage additional values', () => {
+    beforeEach(() => {
+      _this.props.bgImage = 'https://sameimage.com/url....';
+    });
+    it('should assign bgDarken value to backgroundImage opacity', () => {
+      const bgDarkenValue = 0.5;
+      const backgroundImageValue = `linear-gradient( rgba(0, 0, 0, ${bgDarkenValue}), rgba(0, 0, 0, ${bgDarkenValue}) ), url(${
+        _this.props.bgImage
+      })`;
+      _this.props.bgDarken = bgDarkenValue;
+      const res = fn();
+      expect(res.backgroundImage).toEqual(backgroundImageValue);
+    });
+
+    it('should assign bgSize value to backgroundSize', () => {
+      const bgSizeValue = 'auto';
+      _this.props.bgSize = bgSizeValue;
+      const res = fn();
+      expect(res.backgroundSize).toEqual(bgSizeValue);
+    });
+
+    it("should assign 'cover' to backgroundSize", () => {
```

Comment:
You'll get an ESLint `quotes` violation here and on L190, so just disable it line by line, i.e. `eslint-disable-next-line quotes`.